aixplain/modules/pipeline/designer/__init__.py

```diff
@@ -0,0 +1,68 @@
+from .nodes import (
+    Asset,
```

Comment:
Maybe find another name for this class to avoid ambiguity? Like `AssetNode`

Reply:
done